src/slices/MessagesSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
```

Comment:
Oh also. can you write this file in typescript? :) we're trying to slowly introduce typescript into our codebase.

Reply:
Yup

Reply:
Done